bookkeeper-server/src/test/java/org/apache/bookkeeper/client/BookKeeperTest.java

```diff
@@ -818,4 +818,68 @@ public void testReadEntryReleaseByteBufs() throws Exception {
             }
         }
     }
+
+    /**
+     * Tests that issuing multiple reads for the same entry at the same time works as expected.
+     *
+     * @throws Exception
+     */
+    @Test(timeout = 20000)
```

Comment:
Remove timemout? 
I recall there was a require to remove all the "timeout", since we have a global timeout set in surefire configuration.

Reply:
Sure, will update